docs/source/upgrade-guide/index.md

```diff
@@ -170,6 +170,22 @@ In your add-ons and projects, we advise you to always use the public components
 
 This change improves UX of the Babel view (translation form) since a disabled field cannot be selected to be copied over.
 
+### `volto-slate` Cypress helpers moved to its own module
+
+There were some Cypress helpers for `volto-slate` along with the other definition of Cypress commands.
+The Cypress command definitions are intended to be only loaded once, and the helpers can be imported a number of times.
+Therefore, we moved the helpers to its own module:
+
+```
```

Comment:
Is this JavaScript (js) or React (jsx)? I'm guessing .js based on the `commands.js` file name.

````suggestion
```js
````